vendor/github.com/containerd/containerd/go.mod

```diff
@@ -29,41 +29,42 @@ require (
 	github.com/emicklei/go-restful v2.9.5+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/gogo/googleapis v1.4.0
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.4.2
-	github.com/google/go-cmp v0.5.1
-	github.com/google/uuid v1.1.1
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
```

Comment:
this one has a replace-rule so shouldn't updated; https://github.com/containerd/containerd/blob/19ee068f93c91f7b9b2a858457f1af2cabc7bc06/go.mod#L72